### PR TITLE
Created menu and modal components

### PR DESCRIPTION
**Pull request by sethduffin**

### CODE REVIEW

File: client/src/components/organisms/GridContainer/GridContainer.tsx [hunk 1]
```diff
@@ -14,28 +19,139 @@ export interface ButtonProps extends React.PropsWithChildren<any> {
...
+    const modalActionButtons = (
+      <div>
+        <Button className="manage-cards__modal__button">
+          <i className="material-icons">add</i>
+          <span>New Card</span>
+        </Button>
+      </div>
+    );
+
+    const listData = children.map((item) => ({
```
Comment (TannerHolladay) [inline client/src/components/organisms/GridContainer/GridContainer.tsx:81]: I think we should use React.Children.map here. Doing it this way causes the code to break if there is only one child.

File: client/src/App.css [hunk 2]
```diff
@@ -1,3 +1,11 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap');
+@import url('https://fonts.googleapis.com/icon?family=Material+Icons');
+@import url('https://fonts.googleapis.com/icon?family=Material+Icons+Outlined');
+
+:root {
+  font-family: 'Lato', sans-serif;
+};
```
Comment (TannerHolladay) [inline client/src/App.css:7]: Not a big issue, but a semicolon isn't needed here

**Review by TannerHolladay — APPROVED** [review]
Just fix the one issue, and you're good to merge!

File: client/src/components/organisms/GridContainer/GridContainer.tsx (hunk 1 above)
Comment (sethduffin) [reply client/src/components/organisms/GridContainer/GridContainer.tsx:81]: Oh cool, I didn't know about that, thanks

File: client/src/App.css (hunk 2 above)
Comment (sethduffin) [reply client/src/App.css:7]: Good catch

**Pull request merged**